server/admin_management/api/app/repositories/application_admin_repository.py

```diff
@@ -65,3 +66,26 @@ def delete_application_admin(self, application_admin_id: int):
         )
         self.db.delete(record)
         self.db.flush()
+
+    def get_user_app_admin_grants(self, user_id: int) -> List[FamApplication]:
+        """
+        Find out from `app_fam.fam_application_admin` the applications
+            being granted for the user; including "FAM" application.
+
+        Filter on: Only 'IDIR' type user can be an Application Admin.
```

Comment:
I think our create app admin don't check if user type is IDIR or not when creating, maybe we need to create a ticket to add that check.

Reply:
This is in our list of stories for the Delegated Admin theme.

Reply:
okay, so I created a new ticket #1175 to add the check on user type when creating the application admin

Reply:
Yes, thanks for thought of that. Frontend probably is fine but we should add this check at the backend. Thanks.

Reply:
The criteria of removing this filter has been added into #1175 (so this filtering will be removed when #1175 is implemented).